integration-tests/helpers/action.ts

```diff
@@ -0,0 +1,40 @@
+import {driver} from "@wdio/globals";
+interface ActionButtons {
```

Comment:
I'm not sure the overhead of this abstraction is worthwhile, I think it'd be enough to have tests call `pressButton("foo")` directly